website/docs/components/icon/partials/code/how-to-use.md

```diff
@@ -0,0 +1,123 @@
+## How to use this component
+
+The most basic invocation of the Icon component requires only the `@name` property to be passed with a value matching an existing name in [the Icon library](/icons/library):
+
+```handlebars
+<Hds::Icon @name="alert-circle" />
+```
+
+It renders to this (where the `id` will be unique each time):
+
+```markup
+<svg
```

Comment:
There was a comment about updating this code snippet, but was probably lost. 

See suggestions below